actors/miner/src/types.rs

```diff
@@ -345,8 +345,8 @@ pub struct SectorOnChainInfo {
     pub expected_day_reward: TokenAmount,
     /// Expected twenty day projection of reward for sector computed at activation time
     pub expected_storage_pledge: TokenAmount,
-    /// Age of sector this sector replaced or zero
-    pub replaced_sector_age: ChainEpoch,
+    /// Epoch at which this sector's power was most recently updated
+    pub power_base_epoch: ChainEpoch,
     /// Day reward of sector this sector replace or zero
     pub replaced_day_reward: TokenAmount,
```

Comment:
The comment and name are out of date too. The name could possibly stay, but the comment should be something like "Day reward of this sector before its power was most recently updated".